src/scripts/macros/skill-save-prompt/helpers.ts

```diff
@@ -0,0 +1,71 @@
+import { ActorPF2e, CharacterPF2e } from "@actor";
+import { calculateSimpleDC, DCAdjustment } from "@module/dc.ts";
+import { CharacterSkill } from "@actor/character/types.ts";
+import { PROFICIENCY_RANKS } from "@module/data.ts";
+
+const dcAdjustmentsLabel = new Map<DCAdjustment, string>([
```

Comment:
This is already in src/module/dc.ts. Either try to find a way to use the functions in there, or export the structure that's already in there. Ideally the first one, the second one is last resort.